tests/test_pytorch_sparse_dataloader_api.py

```diff
@@ -0,0 +1,249 @@
+
+    def test_except_with_diff_number_of_x_y_sparse_rows(self):
+        with tempfile.TemporaryDirectory() as tiledb_uri_x, tempfile.TemporaryDirectory() as tiledb_uri_y:
+            # Add one extra row on X
+            dataset_shape_x = (ROWS + 1, INPUT_SHAPES[0])
+            dataset_shape_y = (ROWS, (NUM_OF_CLASSES,))
+
+            ingest_in_tiledb_sparse(
+                uri=tiledb_uri_x,
+                data=create_sparse_array_one_hot_2d(
+                    dataset_shape_x[0], dataset_shape_x[1]
+                ),
+                batch_size=BATCH_SIZE,
+            )
+            ingest_in_tiledb_sparse(
+                uri=tiledb_uri_y,
+                data=create_sparse_array_one_hot_2d(
+                    dataset_shape_y[0], dataset_shape_y[1]
+                ),
+                batch_size=BATCH_SIZE,
+            )
+
+            with tiledb.open(tiledb_uri_x) as x, tiledb.open(tiledb_uri_y) as y:
+                with self.assertRaises(Exception):
+                    PyTorchTileDBSparseDataset(
+                        x_array=x, y_array=y, batch_size=BATCH_SIZE
+                    )
+
+    # This test is practically skipped but is linked to the TODO#1
+    def test_sparse_no_duplicates_with_multiple_workers(self):
+        for workers in NUM_OF_WORKERS[2:]:
+            with self.subTest():
+                with tempfile.TemporaryDirectory() as tiledb_uri_x, tempfile.TemporaryDirectory() as tiledb_uri_y:
+
+                    dataset_shape_x = (ROWS,) + INPUT_SHAPES[1]
+                    dataset_shape_y = (ROWS, (NUM_OF_CLASSES,))
+
+                    ingest_in_tiledb_sparse(
+                        uri=tiledb_uri_x,
+                        data=create_sparse_array_one_hot_2d(
+                            dataset_shape_x[0], dataset_shape_x[1]
+                        ),
+                        batch_size=BATCH_SIZE,
+                    )
+                    ingest_in_tiledb_sparse(
+                        uri=tiledb_uri_y,
+                        data=create_sparse_array_one_hot_2d(
+                            dataset_shape_y[0], dataset_shape_y[1]
+                        ),
+                        batch_size=BATCH_SIZE,
+                    )
+
+                    with tiledb.open(tiledb_uri_x) as x, tiledb.open(tiledb_uri_y) as y:
+
+                        tiledb_dataset = PyTorchTileDBSparseDataset(
+                            x_array=x, y_array=y, batch_size=BATCH_SIZE
+                        )
+
+                        self.assertIsInstance(
+                            tiledb_dataset, torch.utils.data.IterableDataset
+                        )
+
+                        train_loader = torch.utils.data.DataLoader(
+                            tiledb_dataset, batch_size=None, num_workers=workers
+                        )
+
+                        unique_inputs = []
+                        unique_labels = []
+
+                        for batchindx, data in enumerate(train_loader):
```

Comment:
Can assign `x, y = data[0].numpy(), data[1].numpy()` and use `x,y` below

Reply:
The test will be removed. Thanks